.devcontainer/cpp/test/testsuite.bats

```diff
@@ -205,3 +205,15 @@ function build_and_run_with_sanitizers() {
   assert_failure
   assert_output --partial "runtime error: load of null pointer"
 }
+
+@test "using clang-uml to generate uml class diagram" {
```

Comment:
Can you keep this in the same vein as the other tests?

I.e. "using clang-uml should generate a uml class diagram for source file".